components/metadata/_metadata.scss

```diff
@@ -0,0 +1,36 @@
+/* _metadata */
+.metadata {
+
+ dd {
+   margin-left: 0; /* As normalize adds a 40px left margin */
+ }
+ &__value,
+ &__term {
+   margin: 0 0 0.3rem;
+ }
+
+ &__list {
+   margin: 0;
+ }
+
+}
+
+/* Message variant */
+.metadata--message {
```

Comment:
I normally avoid naming conventions which are cross component. I'd think about what this variant is actually doing rather than it's implementation.

Reply:
@boxadesign this wasn't in use in the .hbs for `message` so has been removed too.